ui/src/components/PulseBindings/index.jsx

```diff
@@ -50,6 +50,7 @@ import DeleteIcon from 'mdi-react/DeleteIcon';
 }))
 export default class PulseBindings extends Component {
   static propTypes = {
+    patternName: string,
```

Comment:
Why do we need a `patternName` prop?

Reply:
in PulseMessages component, for bindings `pattern` key is used so I am passing patternName as input to component, if there is no name passed it will default it to `pattern`.
As on addBinding function call, uses this name as key and same key is used while rendering

Reply:
I was planning to look at this once the rebase is done. Right now there's some merge conflict in this file so I didn't bother looking at it. I plan on looking at it after the rebase :)